readme.md

```diff
@@ -28,35 +28,19 @@ Please make sure you install the following tools before starting with the instal
 
 ## Installation
 
-> Note that you're free to adjust the `~/Sites/pastebin` location to any directory you want on your machine.
+> Note that you're free to adjust the `~/Code/pastebin` location that will be in the `Homestead.yaml` to any directory you want on your machine.
 
 1. Clone this repository: `git clone git@github.com:laravelio/pastebin.git pastebin`
-2. Add the `Homestead.yaml` file from below to the root of your project
+2. Run `./vendor/bin/homestead make`
 3. Run `vagrant up`
-4. SSH into your Vagrant box, go to `/home/vagrant/pastebin` and run the following commands:
+4. SSH into your Vagrant box, go to `/home/vagrant/Code/pastebin` and run the following commands:
     1. `composer install`
-    2. `php artisan migrate`
-5. Add `192.168.10.30 pastebin.app` to your computer's `/etc/hosts` file
-
-```yaml
-ip: 192.168.10.30
-memory: 2048
-cpus: 1
-provider: virtualbox
-authorize: ~/.ssh/id_rsa.pub
-keys:
-    - ~/.ssh/id_rsa
-folders:
-    - { map: ~/Sites/pastebin, to: /home/vagrant/pastebin }
-sites:
-    - { map: pastebin.app, to: /home/vagrant/pastebin/public }
-databases:
-    - homestead
-name: pastebin
-hostname: pastebin
-```
-
-You can now visit the app in your browser by visiting [http://pastebin.app/](http://pastebin.app).
+    2. `cp .env.example .env`
```

Comment:
This is unnecessary as it's done with a Composer hook. Check the `composer.json` file.